gulp/config.js

```diff
@@ -8,7 +8,7 @@ export const useHandlebars = true;
  */
 export const generateBanners = false;
```

Comment:
Does it make sense to expose  `generateBanners` and `useHandlebars` settings as well to `baumeister.json`?

Not sure though, because the other settings are related to »adding new dependencies« while  `generateBanners` and `useHandlebars` are settings you choose once per project. 🤔